deployment.rst

```diff
@@ -54,7 +54,8 @@ Using Source Control
 If you're using source control (e.g. Git or SVN), you can simplify by having
 your live installation also be a copy of your repository. When you're ready
 to upgrade it is as simple as fetching the latest updates from your source
-control system.
+control system. A common approach for this in Git is, to tag your release and
+check out the Git tag on deployment (see `Git Tagging`_).
```

Comment:
@jepster is tag checkout more common or popular than pull master branch? I think it depend on methodology. I met both approach and I can't say that one of them is much more popular.